crates/bevy_render/src/camera/projection.rs

```diff
@@ -161,17 +161,49 @@ pub enum ScalingMode {
     FixedHorizontal(f32),
 }
 
+/// Project a 3D space onto a 2D surface using parallel lines, i.e., objects have the same
+/// apparent size regardless of depth. The volume contained in the projection is called the view frustum.
```

Comment:
What does "apparent size" mean here in the phrase "objects have the same apparent size"? Is it sufficient to just say "objects have the same size"?

Reply:
It's probably obvious, but I thought I'd specify that perspective only the size objects appear and not their actual size.

Reply:
@xgbwei Ah I see, makes a lot of sense. Would you mind putting a parenthetical explanation then for apparent size? It was not immediately obvious to me.

Something like:

> objects have the same apparent size (the size at which they appear on screen, not their actual size) regardless of depth

Reply:
> @xgbwei Ah I see, makes a lot of sense. Would you mind putting a parenthetical explanation then for apparent size? It was not immediately obvious to me.

I think it'd be better to rephrase to "The size at which objects appear remains the same regardless of depth". The sentence is already behind an 'i.e.' so another parenthesis might be a bit much.

Reply:
Good eye, I agree.